packages/website/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+
+import { Box } from '@coderscamp/ui/components/Box';
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { Logo } from '@coderscamp/ui/components/Logo';
+import { Typography } from '@coderscamp/ui/components/Typography';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+import {
+  SolidFacebookIcon,
+  SolidGitHubIcon,
+  SolidGlobalIcon,
+  SolidInstagramIcon,
+  SolidLinkedinIcon,
+} from '@coderscamp/ui/icons';
+import { colors } from '@coderscamp/ui/theme/overwrites/foundations/colors';
+
+export const socialMediaIcons = [
+  { name: 'FacebookIcon', component: <SolidFacebookIcon /> },
+  { name: 'InstagramIcon', component: <SolidInstagramIcon /> },
+  { name: 'LinkedinIcon', component: <SolidLinkedinIcon /> },
+  { name: 'GitHubIcon', component: <SolidGitHubIcon /> },
+  { name: 'GlobalIcon', component: <SolidGlobalIcon /> },
+];
+
+export const Footer = () => {
+  const footerStyles = {
+    upperFooter: {
+      direction: useBreakpointValue({ base: 'column', lg: 'row' } as const),
+    } as const,
+    lowerFooter: {
+      direction: useBreakpointValue({ base: 'column', md: 'row' } as const),
+      order: useBreakpointValue({ base: -1, md: 1 } as const),
+    } as const,
+  } as const;
+
+  return (
+    <Flex w="100%" direction="column" align="center" bg={colors.gray['50']} color={colors.gray['500']} fontSize="16px">
```

Comment:
💭 When you use colors you can write them simply as `"gray.50"`. Importing `colors` directly could be problematic as we cannot change those colors dynamically

Reply:
I'm not sure If I understand what's different, but I'll make like you told

Reply:
But why we have "colors.ts" file if we shouldn't use it?

Reply:
When you write strings you use values from the theme object that is provided by context. In such a case when we change the theme (dynamically) all color tokens will be replaced by different values. It wouldn't be possible with a plain `colors` object.

colors.ts is a file when we define all colors accessible through the theme. You can track where they are imported and you will find our ThemeProvider